app/models/migration/ecf/base_record.rb

```diff
@@ -0,0 +1,11 @@
+module Migration::Ecf
```

Comment:
Hi @ethax-ross what do you think about adding `inflect.acronym "ECF"` to the inflections file as we have in https://github.com/DFE-Digital/early-careers-framework/blob/main/config/initializers/inflections.rb#L25 so module name can be in capital letter? `Migration::ECF`

Reply:
yeah I think Pete mentioned this too, happy to change it here 👍

Reply:
actually, its going to change a lot of existing code (the `Ecf` namespace is already in the app for other things) so lets leave that for a separate PR